querybook/server/lib/result_store/stores/db_store.py

```diff
@@ -32,9 +30,10 @@ def _get_first_n_lines(self, n: Optional[int]) -> List[str]:
     def get_csv_iter(
         self, number_of_lines: Optional[int] = None
     ) -> Generator[List[List[str]], None, None]:
-        curr_lines = self._get_first_n_lines(number_of_lines)
-        raw_csv_str = "\n".join(curr_lines)
-        return str_to_csv_iter(raw_csv_str)
+        csv_iter = str_to_csv_iter(self.read_raw())
+        if number_of_lines is None:
```

Comment:
I think if the second arg of `islice` is None, it'll return the entire iterator, so this if statement can be removed